src/include/distributed/pg_dist_partition.h

```diff
@@ -52,6 +52,7 @@ typedef FormData_pg_dist_partition *Form_pg_dist_partition;
 #define DISTRIBUTE_BY_APPEND 'a'
 #define DISTRIBUTE_BY_HASH 'h'
 #define DISTRIBUTE_BY_RANGE 'r'
+#define DISTRIBUTE_BY_ALL 'l'
```

Comment:
Alternatively we could start using the enum directly, that'd certainly be easier to read from SQL. Albeit a bit more annoying from C.